governance/votes/Executive vote - September 13, 2023.md

```diff
@@ -0,0 +1,174 @@
+
+**Voting for this executive proposal will place your MKR in support of the actions outlined above.**
+
+Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the actions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+For voters' convenience, each proposed item is prefaced with information showing its provenance. 
+
+- "Authorization" links to the official document(s) or governance action authorizing the proposed item's inclusion in the Executive Vote. This can be the Atlas, a Scope Alignment Artifact, a successful governance poll and/or a ratified budget proposal. 
+- "Reasoning" links to the MakerDAO Forum post providing the rationale and additional context for each item. 
+- "Authorizing Parties" may also be listed (with or without links). These may include the responsible Scope Facilitator and/or Scope Advisory Council members.
+
+### Stability-Scope Defined Parameter Changes
+
+*Authorization*: [Stability Scope Artifact](https://mips.makerdao.com/mips/details/MIP104#0-the-stability-scope)\
+*Reasoning*: [Forum post](https://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969/1) "Stability Scope Parameter Changes"\
+*Authorizing Party:* [Stability Scope Facilitator](http://forum.makerdao.com/t/stability-scope-parameter-changes-5/21969/4)
+
+The Scope-defined parameter changes below have been proposed by Stability Scope Advisory Council Member [Block Analitica (BA Labs)](https://forum.makerdao.com/t/professional-ecosystem-actor-introduction-ba-labs/20813): 
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Changes
+
+- ETH-A Stability Fee will be increased by 0.12% from 3.58% to **3.70%**.
+- ETH-B Stability Fee will be increased by 0.12% from 4.08% to **4.20%**.
+- ETH-C Stability Fee will be increased by 0.12% from 3.33% to **3.45%**.
+
+#### PSM-USDP-A DC-IAM [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Change
+
+- Maximum Debt Ceiling will be increased from 0 (zero) to **120 million DAI**.
+
+### Spark Protocol Dai Direct Deposit Module (D3M) Parameter Changes
+
+*Authorization*: [Governance Poll 2023-09-04](https://vote.makerdao.com/polling/QmQnUhZt#poll-detail)\
+*Reasoning*: [Forum post](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801) "Upcoming Spell Proposed Changes"\
+*Authorizing Parties:* [Stability Scope Facilitator](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801/4); [BA Labs](http://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801/5)
```

Comment:
The Authorizing party is governance voters who have approved the poll. As written it implies that the Scope Facilitator has pushed this to the exec level which is not the case. The Facilitator approved the poll creation. BA Labs do not have any governance mandate for authorizing these changes at either poll or exec level